src/components/featured-item/featured-item-print.scss

```diff
@@ -31,13 +31,22 @@ $featured-item-print: null !default;
 }
 
 .ecl-featured-item__title {
-  margin: map.get($featured-item-print, 'title-spacing-desktop');
+  margin-bottom: map.get($featured-item-print, 'title-spacing-desktop-bottom');
```

Comment:
it feels like this went a bit beyond the initial expectations, i know that this css was the results of dozens of iterations which messed it up, but now we get lot of changes in this pr which was supposed to be a one line fix. What i would do, at this point, is to use margin-block-start and margin-block-end instead of the old margin top and bottom, it has not a direct impact like the inline margins but it's how it should be.
To save some writing, though, when you define both the inline or the block spacings, you can use a single line shorthand, like here it would be margin-block: map.get($featured-item-print, 'title-spacing-desktop-top') map.get($featured-item-print, 'title-spacing-desktop-bottom');